packages/react-components/src/components/ControlBarButton.tsx

```diff
@@ -55,16 +63,6 @@ export interface ControlBarButtonProps extends IButtonProps {
   onRenderOffIcon?: IRenderFunction<IButtonProps>;
 }
 
-const DefaultRenderText = (props?: ControlBarButtonProps): JSX.Element => {
-  const labelText =
-    props?.text ?? props?.strings?.label ?? (props?.checked ? props?.strings?.onLabel : props?.strings?.offLabel);
-  return (
-    <Label key={props?.labelKey} className={mergeStyles(controlButtonLabelStyles, props?.styles?.label)}>
```

Comment:
I had to refactor this to the callback on line 82 because for some reason `props?.strings?.label` has some default styling including a font weight of 600 as shown below.
![image](https://user-images.githubusercontent.com/79475487/133392118-f3bec064-e6ce-4704-9db8-64059c793740.png)
![image](https://user-images.githubusercontent.com/79475487/133392568-c69bcdd7-b857-4aae-8a56-2bb841e68ae4.png)